### PR TITLE
[Doc Improvement][Update task module to dialog glossary and faq][3480222]

### PR DESCRIPTION
**Pull request by v-npaladugu**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/get-started/glossary.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d1c6316e7225898ef62558afe7acc9c831e9db55/msteams-platform%2Fget-started%2Fglossary.md) | 92 | 66 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/dcb9f5e3-1deb-44d6-8050-8925e736b6ce) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/d1c6316e7225898ef62558afe7acc9c831e9db55/msteams-platform%2Fmstdd-landing.yml) | 92 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5795f6e0-db40-4a9b-bcd1-eb11653a9e66) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/teams-faq.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d1c6316e7225898ef62558afe7acc9c831e9db55/msteams-platform%2Fteams-faq.md) | 90 | 74 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bf396599-899a-4e97-9239-a0bea39603a4) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/get-started/glossary.md [hunk 1]
```diff
@@ -254,8 +255,8 @@ Common terms and definitions used in Microsoft Teams developer documentation.
 | --- | --- |
 | [Tab](../tabs/what-are-tabs.md) | Tabs are client-aware webpages embedded in Microsoft Teams, Outlook, and Microsoft 365 that point to domains declared in app manifest. You can add it as part of a channel inside a team, group chat, or personal app for an individual user. |
 | [Tab chat](../tabs/how-to/conversational-tabs.md) | A type of tab that lets a user have a focused conversation experience in dynamic tabs. |
-| [Task modules](../task-modules-and-cards/what-are-task-modules.md) | A feature of Teams app to create modal pop-up for completing tasks, displaying videos, or dashboard. <br> **See also**: [Adaptive Card](#a) |
-| [Task info](../task-modules-and-cards/task-modules/invoking-task-modules.md#the-taskinfo-object) | The `TaskInfo` object contains the metadata for a task module.|
+| [Task modules](../task-modules-and-cards/what-are-task-modules.md) (referred as dialogs in TeamsJS v.2.0)| A feature of Teams app to create modal pop-up for completing tasks, displaying videos, or dashboard. <br> **See also**: [Adaptive Card](#a) |
```
Comment (erikadoyle) [inline msteams-platform/get-started/glossary.md:258]: Same thing for v2--could we instead say "TeamsJS v2.x" to cover all minor versions?

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/get-started/glossary.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1d454f2bbb9cadafb565e10d38112caa550ac527/msteams-platform%2Fget-started%2Fglossary.md) | 92 | 66 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a9956164-779e-41cf-933c-1361aabaee89) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/1d454f2bbb9cadafb565e10d38112caa550ac527/msteams-platform%2Fmstdd-landing.yml) | 92 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5d95ce16-63e0-45cf-ba0a-94e9160a7873) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/teams-faq.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1d454f2bbb9cadafb565e10d38112caa550ac527/msteams-platform%2Fteams-faq.md) | 90 | 74 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/540651b0-0c23-41e0-b26a-af5bd6b7e512) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/get-started/glossary.md (hunk 1 above)
Comment (v-npaladugu) [reply msteams-platform/get-started/glossary.md]: Updated

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/get-started/glossary.md](https://github.com/MicrosoftDocs/msteams-docs/blob/596f223bc431de7dab3527fc6c84970726d86d6f/msteams-platform%2Fget-started%2Fglossary.md) | 92 | 66 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/40b8c4c3-77da-4641-9fda-7971127dadb5) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/596f223bc431de7dab3527fc6c84970726d86d6f/msteams-platform%2Fmstdd-landing.yml) | 92 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/495ad717-c0dd-43c6-abf7-c0be6928a79a) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/teams-faq.md](https://github.com/MicrosoftDocs/msteams-docs/blob/596f223bc431de7dab3527fc6c84970726d86d6f/msteams-platform%2Fteams-faq.md) | 90 | 74 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/acef83e5-54f7-40e5-8e54-f964c360f6a0) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**